src/lib/web-llm-helper.ts

```diff
@@ -1,52 +1,64 @@
 // https://github.com/mlc-ai/web-llm/blob/main/examples/get-started-web-worker
 
 import useChatStore from "@/hooks/useChatStore";
-import * as webllm from "@mlc-ai/web-llm";
 import { Model } from "./models";
 import { Document } from "@langchain/core/documents";
 import { getEmbeddingsInstance } from "./embed";
+import type { AppConfig, MLCEngineInterface, InitProgressReport, CreateWebWorkerMLCEngine} from "@mlc-ai/web-llm";
```

Comment:
using `import type` so it's only for type checking and not part of the actual imports